scripts/dev-e2e.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+set -eo pipefail
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+STATUSFILE='armadactl-query.txt'
+AOREPO='https://github.com/armadaproject/armada-operator.git'
+AOHOME=../armada-operator
```

Comment:
Should not assume CWD is root of armada-spark repo:
```suggestion
AOHOME=."$scripts/../../armada-operator"
```

Reply:
Good catch - fixed in commit 98406ac.